linters/rubocop/rubocop_to_sarif.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+
+import json
+import sys
+
+
+def map_severity(severity):
+    if severity in ["convetion", "refactor", "info"]:
```

Comment:
Looks like there might be a typo here:

```suggestion
    if severity in ["convention", "refactor", "info"]:
```

Reply:
Thanks!